### PR TITLE
Lazy-load PDF and ZIP libraries in the client

### PR DESCRIPTION
**Pull request by 7sempra**
These libraries are quite large, so we should only load them
when we need them.

### CODE REVIEW

File: client/src/ui/deckbuilder/DeckBuilderExportMenu.vue [hunk 1]
```diff
@@ -56,34 +68,48 @@ export default Vue.extend({
     },
 
     exportedDeck(): string {
-      return this.deck ? deckToXml(this.deck) : '';
+      return this.deck ? exportLoader.chunk.deckToXml(this.deck) : '';
```
Comment (kprevas) [inline client/src/ui/deckbuilder/DeckBuilderExportMenu.vue:71]: Don't you have to check libLoaded everywhere you do exportLoader.chunk?

File: client/src/ui/deckbuilder/DeckBuilderExportMenu.vue [hunk 2]
```diff
@@ -56,34 +68,48 @@ export default Vue.extend({
...
 
 <style scoped>
 ._deck-builder-export-menu {
-  background: white;
-  box-shadow: #a5a5a5 3px 2px 2px;
+  background-color: #FFF;
+  border-radius: 5px;
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+  border: 1px solid #ccc;
+  min-width: 150px;
```
Comment (kprevas) [inline client/src/ui/deckbuilder/DeckBuilderExportMenu.vue:112]: I fixed this in my PR

File: client/src/ui/deckbuilder/DeckBuilderExportMenu.vue (hunk 1 above)
Comment (7sempra) [reply client/src/ui/deckbuilder/DeckBuilderExportMenu.vue:71]: These computed values are only called when necessary and everything that requires them is wrapped in the <template> that checks if the lib is loaded, so we should be okay.

File: client/src/ui/deckbuilder/DeckBuilderExportMenu.vue (hunk 2 above)
Comment (7sempra) [reply client/src/ui/deckbuilder/DeckBuilderExportMenu.vue:112]: Nice! Updating the values a little to match the existing popovers.

**Pull request merged**